### PR TITLE
feat: Authentication docs

### PR DESCRIPTION
**Pull request by gunnar2k**

### CODE REVIEW

File: src/authentication/README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+```
+
+For subsequent calls, use the value in `access_token` in the `Authorization` field. 
+
+Here is an example:
+
+```bash
+curl https://api.us.vic.ai/v0/healthCheck \
+    -H "Content-Type: application/json" \
+    -H "Authorization: YOUR_ACCESS_TOKEN"
```
[the remark this answers is not in the document]
Comment (mbaleczny) [reply src/authentication/README.md:44]: or like this? 🤔 

```suggestion
    -H "Authorization: TOKEN_TYPE YOUR_ACCESS_TOKEN"
```

[the remark this answers is not in the document]
Comment (gunnar2k) [reply src/authentication/README.md:44]: I dont think thats accurate tho, look at any of the tests for public api controllers, e.g test/public_api/web/v0/controllers/vendor_tag_controller_test.exs:403 - we dont seem to set it with the "Bearer" prefix

**Review by mbaleczny — APPROVED** [review]

File: src/authentication/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (warmwaffles) [reply src/authentication/README.md:44]: This is incorrect. `Bearer` should ALWAYS be present.

[the remark this answers is not in the document]
Comment (gunnar2k) [reply src/authentication/README.md:44]: Ya I just realized the `Bearer` prefix is optional `lib/public_api/auth_provider/bearer.ex` 🙄

[the remark this answers is not in the document]
Comment (warmwaffles) [reply src/authentication/README.md:44]: We should enforce it. Set a deadline of July 1st and require it.

**Pull request merged**